pkg/api/file_repository.go

```diff
@@ -16,6 +16,11 @@ type FileSystemRepository struct {
 	sync.RWMutex
 }
 
+// Type used for  JSON.Unmarshaller
+type definitionList struct {
+	defs []Definition
```

Comment:
I believe this should be a slice of definition pointers? otherwise a lot of coping will happen